examples/cpp/README.md

```diff
@@ -1,5 +1,21 @@
-# How to build
+# How to use
```

Comment:
for me, they are still build instructions.

Reply:
I'd say that the title of the 'Python instructions' document is not great, and in the future, if you add a _"create&use a container that cross-compile it_" instruction, you would need to create a different subsection anyway, then change the title when it makes sense.